eplaunch/workflows/manager.py

```diff
@@ -0,0 +1,44 @@
+# import importlib
+from importlib import util as import_util
+import inspect
+import os
+import re
+
+
+def get_workflows(workflow_path=None):
+    this_file_directory_path = os.path.dirname(os.path.realpath(__file__))
+    built_in_workflow_directory = os.path.join(this_file_directory_path, 'builtin', 'workflows')
+
+    if not workflow_path:
+        workflow_path = built_in_workflow_directory
+
+    py_search_regex = re.compile('.py$', re.IGNORECASE)
+    workflow_files = filter(py_search_regex.search, os.listdir(workflow_path))
+
+    modules = []
+    for this_file in workflow_files:
+        if '__init__.py' in this_file:
```

Comment:
We loop through all files found, except the __init__.py files -- maybe we don't need these files anyway now, but it should be OK to ignore regardless.